coalib/parsing/LineParser.py

```diff
@@ -56,6 +58,17 @@ def parse(self, line):
             line,
             self.comment_separators)
         comment = unescape(comment)
+
+        if line.startswith('#'):
+            if(line[1] != ' '):
+                logging.warning('Your line has no whitespace after #')
+
+        if (line != '' and line.find('#') != -1):
+            pos = [i for i, x in enumerate(line) if x == '#']
```

Comment:
Also, the variable names `pos`, `i`, and `x` are not very descriptive. I'd suggest `comment_start` (if we'll only get the first `#` character), `index`, and `char`.